From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: minotaa <jpichardo369@gmail.com>
Date: Sun, 1 Jun 2025 11:29:13 -0400
Subject: [PATCH] Rebrand 2


diff --git a/build.gradle.kts b/build.gradle.kts
index c1195aa000515f33f4909123cfeaff5603b4cba6..923205e72543669e3235cf40a2d68aad34e7f7b3 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -1,5 +1,5 @@
 plugins {
-    id("pandaspigot.conventions")
+    id("aerosmith.conventions")
     id("com.gradleup.shadow") version "9.0.0-beta7"
 }
 
@@ -10,7 +10,7 @@ repositories {
 }
 
 dependencies {
-    implementation(project(":pandaspigot-api"))
+    implementation(project(":aerosmith-api"))
 
     // PandaSpigot start - Configuration
     implementation("com.hpfxd.configurate:configurate-eo-yaml:1.0.0") {
@@ -51,7 +51,7 @@ dependencies {
 fun TaskContainer.registerRunTask(
     name: String, block: JavaExec.() -> Unit
 ): TaskProvider<JavaExec> = register<JavaExec>(name) {
-    group = "pandaspigot"
+    group = "aerosmith"
     standardInput = System.`in`
     workingDir = rootProject.layout.projectDirectory.dir(
         providers.gradleProperty("runWorkDir").orElse("run")
@@ -74,7 +74,7 @@ tasks {
             "com.amihaiemil.eoyaml",
             "org.spongepowered.configurate",
             "io.leangen.geantyref",
-        ).forEach { relocate(it, "com.hpfxd.pandaspigot.libs.$it") }
+        ).forEach { relocate(it, "dev.limetta.aerosmith.libs.$it") }
         // PandaSpigot end
 
         val cbLibsPkg = "org.bukkit.craftbukkit.libs"
@@ -108,7 +108,7 @@ tasks {
             attributes(
                 "Main-Class" to "org.bukkit.craftbukkit.Main",
                 "Implementation-Title" to "CraftBukkit",
-                "Implementation-Version" to "git-PandaSpigot-$implementationVersion",
+                "Implementation-Version" to "git-Aerosmith-$implementationVersion",
                 "Implementation-Vendor" to date,
                 "Specification-Title" to "Bukkit",
                 "Specification-Version" to project.version,
