From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: minotaa <jpichardo369@gmail.com>
Date: Sun, 1 Jun 2025 12:36:31 -0400
Subject: [PATCH] I don't even know what changed here but okay


diff --git a/src/main/java/net/minecraft/server/WorldGenCaves.java b/src/main/java/net/minecraft/server/WorldGenCaves.java
index 089e6c42e16a65874dd987881558f0ff2a921cb3..e96f806a40f20b3fc49059818fd793b14507e5d0 100644
--- a/src/main/java/net/minecraft/server/WorldGenCaves.java
+++ b/src/main/java/net/minecraft/server/WorldGenCaves.java
@@ -158,7 +158,7 @@ public class WorldGenCaves extends WorldGenBase {
                                                     }
 
                                                     if (flag3 && chunksnapshot.a(j3, j4 - 1, i4).getBlock() == Blocks.DIRT) {
-                                                        blockposition_mutableblockposition.c(j3, j4 + j * 16, 0, i4 + k * 16);
+                                                        blockposition_mutableblockposition.c(j3 + j * 16, 0, i4 + k * 16);
                                                         chunksnapshot.a(j3, j4 - 1, i4, this.c.getBiome(blockposition_mutableblockposition).ak.getBlock().getBlockData());
                                                     }
                                                 }
