From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: minotaa <jpichardo369@gmail.com>
Date: Sun, 1 Jun 2025 19:01:05 -0400
Subject: [PATCH] Cater to plugins using dumb version checking methods


diff --git a/build.gradle.kts b/build.gradle.kts
index 923205e72543669e3235cf40a2d68aad34e7f7b3..ff2e64d396f1a4946a076ef852bf4f65b802bfdb 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -102,7 +102,7 @@ tasks {
         manifest {
             val git = Git(rootProject.layout.projectDirectory)
             val gitHash = git("rev-parse", "--short=7", "HEAD").getText().trim()
-            val implementationVersion = System.getenv("GITHUB_RUN_NUMBER") ?: "\"$gitHash\""
+            val implementationVersion = System.getenv("GITHUB_RUN_NUMBER") ?: "000"
             val date = git("show", "-s", "--format=%ci", gitHash).getText().trim()
             val gitBranch = git("rev-parse", "--abbrev-ref", "HEAD").getText().trim()
             attributes(
diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
index f069bb9903b7ab699b47d7a28cfd988c30778f78..65dd7b6d747ae84ec8a317f69163738f0bff2041 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -127,7 +127,7 @@ import net.md_5.bungee.api.chat.BaseComponent;
 
 public final class CraftServer implements Server {
     private static final Player[] EMPTY_PLAYER_ARRAY = new Player[0];
-    private final String serverName = "Aerosmith"; // PandaSpigot
+    private final String serverName = "AeroSpigot"; // PandaSpigot
     private final String serverVersion;
     private final String bukkitVersion = Versioning.getBukkitVersion();
     private final Logger logger = Logger.getLogger("Minecraft");
